packages/typespec-test/test/eventgrid_modular/generated/typespec-ts/src/rest/isUnexpected.ts

```diff
@@ -27,7 +28,10 @@ const responseMap: Record<string, string[]> = {
 };
 
 export function isUnexpected(
-  response: PublishCloudEvent200Response | PublishCloudEventDefaultResponse
+  response:
+    | PublishCloudEvent200Response
+    | PublishCloudEvents200Response
```

Comment:
It seems that we have sharedRoute cases in smoke testing and there is no error reported during `npm run build`. But the build error is reported in OpenAI.

Do you know what caused the difference? I guess maybe the `tsconfig.json` is inconsistant.

Reply:
Maybe I remeber wrongly and checking here https://github.com/Azure/autorest.typescript/issues/2013#issuecomment-1801186466